### ISSUE TITLE
Can't hear rain when using weather2 (1.7.10)

### ISSUE THREAD

**Issue by Josh74000** [OreCruncher/DynamicSurroundings#371]
*Mod Version:*
1.0.6.3
*Forge Version:*
1614
*Description:*
When I have weather2 v2.3.20 installed with dynamic surroundings, I can't hear rain, I think it must be something to do with weather2 when there is a storm.

**Comment by Josh74000** [comment]
but when I don't have weather2 installed I can hear rain

**Comment by OreCruncher** [comment]
Yep.  Weather2 is in control of rain splashing and what not.  https://github.com/OreCruncher/DynamicSurroundings/wiki/Compatibility#weather-2

**Comment by Josh74000** [comment]
[CleanWeather v1.1.0 for 1.2.5 (for  a future dynamic surroundings update).zip](https://github.com/OreCruncher/DynamicSurroundings/files/2626160/CleanWeather.v1.1.0.for.1.2.5.for.a.future.dynamic.surroundings.update.zip)
If you want to have a look at the mod file, I've attached a file here
